### DISCUSSION TITLE
Install Had a Problem 2.3.9

### DISCUSSION THREAD

**Discussion by Mfisher925** [Security-Onion-Solutions/securityonion#6479] [Unsupported Versions] [answered]
Hello,

I've had issues installing a Stand Alone Manager node with 2 different releases of Security Onion in VMWare ESXi.  I have yet to install/create the other search and forward nodes.

VM Specs: 4 CPU Cores 2.6Ghz, 16 GB Ram, 300 GB Storage, installed via local datastore .ISO

I've tried both releases 2.3.8 and 2.3.9 both show the same errors related to the Kibana container not launching due to a conflict with a container with the same name.  As well as Wuzah being unresponsive and not registering.

The web interface is not reachable however so-status shows all services "ok"

`Checking Docker status

    Docker --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elastalert -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-filebeat ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-fleet ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-grafana ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-idstools ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-influxdb ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-kratos ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-logstash ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-mysql ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-nginx ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-playbook ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-redis ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-wazuh ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
`

Attached is the sosetup.log, pruned error lines, errors.log is empty.
[journalSECON.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7671154/journalSECON.txt)
[so-setup2.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7671155/so-setup2.txt)
[errorspruned.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7671156/errorspruned.txt)

**Comment by dougburks** [comment, marked as answer]
Strange. When you say `The web interface is not reachable`, does that mean that you can't reach Security Onion Console or anything else on port 443? I definitely see the Kibana issue in the log you provided but I don't think that would prevent Security Onion Console from running. 

Have you double-checked all IP, DNS, and hostname settings? 

Have you tried installing on a different host just to see if you get anything different?

**Reply by Mfisher925** [reply]
Doug,

I had installed it on another physical host with the same configuration settings and it showed the same container error but the console was accessible.

I disabled Wazuh as we decided we wouldn't use it as well as NOT running "so-allow".  I still received the same kibana container errors however after running "so-allow" after install the console is reachable.

Now however, when viewing the grid, the only active node, this manager node, is showing "FAULT" status.  I can't find any reference to the FAULT status in the documentation.  Where could I find the related logs/state explanation?

Thanks,

**Reply by dougburks** [reply]
Check the logs in `/opt/so/log/`:
https://docs.securityonion.net/en/2.3/help.html?highlight=%2Fopt%2Fso%2Flog%2F
https://docs.securityonion.net/en/2.3/directory.html?highlight=%2Fopt%2Fso%2Flog%2F

**Reply by Mfisher925** [reply]
Attached are logs grepped for -i "error" from all logs in /opt/so/log/ and I don't see anything that stands out.  This was a fresh install today, that also only showed the kibana container error in the sosetup.log.

The grid still shows the only node, the stand alone manager in "FAULT" status.  With "Uknown" for Process and RAID status.

Kind of at a loss here. 
[grepped_errors_all_secon_logs.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7790474/grepped_errors_all_secon_logs.txt)
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7790481/sosetup.log)

**Reply by dougburks** [reply]
What happens if you do a simple IMPORT installation as shown at https://docs.securityonion.net/en/2.3/first-time-users.html?

**Reply by Mfisher925** [reply]
The install still prompts that "install had a problem".  These were the only errors in sosetup.log.

` *** Restarting Salt to fix any SSL errors. ***
+log_level: error
+log_level_logfile: error
                  +log_level: error
                  +log_level_logfile: error
                error: HTTP 404: Not Found
[ERROR   ] {'image': {'Time_Elapsed': 0.3039431571960449, 'retcode': 0, 'Layers': {'Pulled': ['dbd47cb9f19c', '1f16dc41604f', '2d473b07cdd5', 'fb10673d4851']}, 'Status': 'Downloaded newer image for soimporttest:5000/security-onion-solutions/so-steno:2.3.91'}}
                error: HTTP 404: Not Found
Errors detected during setup; skipping post-setup steps to allow for analysis of failures.
`
so-status shows this

`Checking Docker status

    Docker --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-filebeat ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-idstools ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-kratos ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-nginx ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-steno ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ DISABLED ]
    so-suricata ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ DISABLED ]
    so-zeek -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ DISABLED ]
`

However when connected to the Security Onion Console, the Grid lists the status as OK.  I haven't imported any PCAPs however.

**Reply by dougburks** [reply]
This is really strange. An IMPORT installation should be very simple and very unlikely to fail, so it seems like it might be somehow related to IP, DNS, or hostname settings or perhaps a problem with the machine itself. Is this still a VM or is it a physical host? If VM, is it possible the VM host is oversubscribed? If physical host, is it possible it is underpowered?

Is there any more context in sosetup.log that might help us understand what's going on?

Have you tried doing an airgap installation to see if that makes any difference?

**Reply by Mfisher925** [reply]
This is a VM on an ESXi host Dell PowerEdge T620, when installing the Stand Alone Manager node, it is the only VM on that host.

When installing the IMPORT version, I used the same IP, DNS, Hostname settings as the Stand Alone Manager that kept failing.  I've also installed the Stand Alone Manager node on a separate physical host and encountered the same "FAULT" state in the GRID.

I have attached the sosetup.log and I'll comb back through it for more clues.
[sosetuplog.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7824938/sosetuplog.txt)

**Reply by Mfisher925** [reply]
Doug, 

We may have figured out what was going on.  My colleague did some troubleshooting and found some switch port settings on our network hardware that may have resolved the issue.  

We had a trunk configured attached to our ESXi host that had 3 allowed vlans and a NATIVE vlan configured on the trunk.  The installation appeared to have network connectivity even when it "had a problem" at install.  The web interface was reachable but the GRID showed "FAULT".  

However after removing the NATIVE vlan from the switch port on our Cisco switch, the install did not prompt "install had a problem" and now the GRID shows the status "OK".

We aren't entirely sure why, but that NATIVE vlan configured may have limited network connectivity somehow on the ESXi host.

We will continue with our distributed deployment and hopefully it all goes well from here.

Thanks for the help!
